src/mmw/js/src/compare/views.js

```diff
@@ -170,6 +172,31 @@ var CompareModificationsPopoverView = Marionette.ItemView.extend({
     }
 });
 
+var CompareDescriptionPopoverView = Marionette.ItemView.extend({
+    // model: ScenarioModel
+    template: compareDescriptionPopoverTmpl,
+    className: 'compare-no-mods-popover',
+
+    templateHelpers: function() {
+        var title = this.model.get('name'),
+            text;
+
+        if (this.model.get('is_pre_columbian')) {
```

Comment:
This could be easily done in the template as well, but I figured it was easier to debug if we kept the logic here.